privacy-configuration/config4_reference.json

```diff
@@ -0,0 +1,60 @@
+{
+    "readme": "This configuration has only tracking link features",
+    "features": {
+        "trackingLinks": {
+            "state": "enabled",
+            "exceptions": [
+                {
+                    "domain": "example2.com",
+                    "reason": "Site breakage"
+                }
+            ],
+            "settings": {
+                "ampLinkFormats": [
+                    "https?:\\/\\/(?:w{3}\\.)?google\\.\\w{2,}\\/amp\\/s\\/(\\S+)",
+                    "https?:\\/\\/\\S+ampproject\\.org\\/v\\/s\\/(\\S+)"
+                ],
+                "ampKeywords": [
+                    "=amp",
+                    "amp=",
+                    "&amp",
+                    "amp&",
+                    "/amp",
+                    "amp/",
+                    ".amp",
+                    "amp.",
+                    "%amp",
+                    "amp%",
+                    "_amp",
+                    "amp_",
+                    "?amp"
+                ],
+                "trackingParameters": [
```

Comment:
those don't seem to be tested yet? Do you plan to add tests later on once implementation is ready? I guess we could add tests now (TDD!!).